README.md

```diff
@@ -245,11 +245,17 @@
 ### Term representation
 
 * <a name="underscores-in-numerics"></a>
-  Add underscores to large numeric literals to improve their readability.
+  Add underscores to base-10 numeric literals that have six or more digits.
```

Comment:
According to my understanding, it makes sense to underscore six digits or more since the port range is from 0-65535. :)